zone/entity.cpp

```diff
@@ -1814,14 +1814,14 @@ void EntityList::DuelMessage(Mob *winner, Mob *loser, bool flee)
 	}
 }
 
-Client *EntityList::GetClientByName(const char *checkname)
```

Comment:
Really an out of scope change here - but we'll let it slide for this one. Cleaner for sure

Reply:
Yeah, sorry. Only changed it because I was using it for this.